gRPC_test/src/main/proto/coffeeService.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+package coffee;
+
+option java_multiple_files = true;
+option java_package = "com.example.coffee";
+option java_outer_classname = "CoffeeProto";
+
+// Enum for coffee types
+enum CoffeeType {
```

Comment:
This example with coffee type and coffee example is totally different from our previous coffee example. We should make sure that nothing is confused later on.

This is a nice example, but cannot be re-used for the active learning implementation.